assets/portfolio.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
I recommend putting portfolio.html in your root of your repository because the url to this page on Github pages will then be: https://shriji96.github.io/Basic-Portfolio/portfolio.html INSTEAD OF https://shriji96.github.io/Basic-Portfolio/assets/portfolio.html